cookbooks/aws-parallelcluster-slurm/recipes/test/mock_munge_key.rb

```diff
@@ -6,6 +6,7 @@
   end
 
   file "#{munge_dir}/.munge.key" do
+    mode '0600'
     content 'munge-key'
```

Comment:
`munge-key` is not a good content for a munge key :D

The content must be binary, and at least 32 bytes long. Please find a way to pass a binary blob to this file.

Reply:
Done! I use the same logic as the rotation script does(which means the config and update process do).